tutorials/01_introductory/lifecycle.py

```diff
@@ -0,0 +1,288 @@
+"""
+=======================
+The Lifecycle of a Plot
+=======================
+
+Going from start to finish of one plot in Matplotlib.
+
+This tutorial aims to show the beginning, middle, and end of a single
+visualization using Matplotlib. We'll begin with some raw data and
+end by saving a figure of a customized visualization. Along the way we'll try
+to highlight some neat features and best-practices using Matplotlib.
+
+.. currentmodule:: matplotlib
+
+.. note::
+
+    This tutorial is based off of
+    `this excellent blog post <http://pbpython.com/effective-matplotlib.html>`_
+    by Chris Moffitt. It was transformed into this tutorial by Chris Holdgraf.
+
+A note on Object-Oriented vs Pyplot
+===================================
+
+Matplotlib has two interfaces. The first is based on MATLAB and uses
```

Comment:
I think we should switch these; if this tutorial and the rest of the docs are going to emphasize the OO interface, then it should be "first".